.github/workflows/test.yml

```
@@ -6,7 +6,7 @@ on:
     - 'release/*'
   pull_request:
 env:
-  CORE_REPO_SHA: c49ad57bfe35cfc69bfa863d74058ca9bec55fc3
+  CORE_REPO_SHA: f4b9ac20023395e077f47831a04a1a6b473c3406 # TODO: Update to main
```

Nit: TODO not needed I believe.

---

Aren't PRs squashed when merging? In that case, a new commit ID is generated.

---

Right, but this SHA is used only for testing. So as long as a commit in core does not cause a build failure in this tox build, this SHA doesn't need to be updated. Actually, I don't believe it needs to be updated in this PR either, because your change isn't dependent on any new commits in core.

---

As mentioned in the PR description, the test code depends on https://github.com/open-telemetry/opentelemetry-python/pull/2101 (httpretty does not provide an IP address)

---

Right, I just meant that the `TODO: Update to main` is not needed. I don't believe we need to update the SHA to main even after this is merged.

---

Now that the core PR is merged, this point is moot, but it would have seemed odd to point to a commit that is not on any branch anywhere (after PR merge). Technically, it wouldn't have been a real problem as AFAIK GH keeps commits that were visible on PRs indefinitely, but still.

```suggestion
  CORE_REPO_SHA: d9c22a87b6bfc5ec332588c764f82c32f068b2c3
```